redfish-core/lib/cable.hpp

```diff
@@ -226,9 +226,10 @@ inline void requestRoutesCableCollection(App& app)
         asyncResp->res.jsonValue["Name"] = "Cable Collection";
         asyncResp->res.jsonValue["Description"] = "Collection of Cable Entries";
 
+        constexpr std::array<std::string_view, 1> interfaces{
```

Comment:
This change needs to be moved to inline void requestRoutesCable(App& app).

https://gerrit.openbmc.org/c/openbmc/bmcweb/+/59537/12/redfish-core/lib/cable.hpp#125

Reply:
I did not follow.
The interfaces array is a local variable within the lambda function defined in the call to BMCWEB_ROUTE.

Reply:
It is not used outside of that function, there is no need to mark it as inline.